integrations/grpc/client.go

```diff
@@ -60,6 +75,7 @@ func newGRPCClientWithProtos(conn *grpc.ClientConn, protos protoFiles) (*grpccli
 	return c, nil
 }
 
+//lint:ignore U1000 Ignore unused to keep this code until we decide if we need it
 func (r *grpcclient) getServiceDescriptor(service string) (*desc.ServiceDescriptor, error) {
```

Comment:
i say kill it now, we can get it from the git history.

Reply:
or you can just add another function to the integration now to list methods on a remote service.

Reply:
removed